index.html

```diff
@@ -218,6 +218,18 @@ <h4 class="t" data-t="export.section.query"></h4>
   <div title="" id="export-raw-dialog" class="t dialog" data-t="[title]export.raw.title">
     <p class="message"></p>
   </div>
+  <<div title="" id="export-standalone-query-dialog" class="t dialog" data-t="[title]export.standalone_query_title">
```

Comment:
it seems like there are superfluous `<` brackets here and in the lines below this one